server/lib/routes/post-metrics.js

```diff
@@ -51,19 +52,23 @@ module.exports = function () {
         }
         ga.write(metrics);
 
-        optionallyLogFlowBeginEvent(req, metrics, requestReceivedTime);
+        optionallyLogFlowEvents(req, metrics, requestReceivedTime);
       });
     }
   };
 };
 
-function optionallyLogFlowBeginEvent (req, metrics, requestReceivedTime) {
+function optionallyLogFlowEvents (req, metrics, requestReceivedTime) {
   if (DISABLE_CLIENT_METRICS_STDERR) {
     return;
   }
 
   var events = metrics.events || [];
-  var hasFlowBeginEvent = events.some(function (event) {
+  var flowEvents = _.filter(events, function(event) {
+    return event.type.indexOf('flow.') === 0;
```

Comment:
...ahhh, and here I get my answer. The event names are changed because we use the `flow.` prefix as an indicator that it is a flow event.

Makes sense, but we still need to take into account the analysis of abandoned flows. Could we maybe change:

* `flow.engage` to `flow.signup.engage` and `flow.signin.engage`?
* `flow.attempt_signup` to `flow.signup.submit`?
* `flow.attempt_signin` to `flow.signin.submit`?